specs/binding-tcp.spec/src/main/scripts/io/aklivity/zilla/specs/binding/tcp/streams/application/rfc793/client.sent.data.reconfigure.delete_create/client.rpt

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
What is `delete_create` scenario?
I was expecting to see 3 scenarios; created, modified, deleted

Reply:
I forgot to delete these from the TCP specs. Correcte.